src/state/selectors.js

```diff
@@ -124,3 +127,10 @@ export const getSelectedTopic = createSelector(
     return null;
   }
 );
+
+export const getFieldsDictionary = createSelector(
+  getFieldsData,
+  (data) => {
+    log(data);
+    return data;
+  });
```

Comment:
Do you expect this to change in the future? Might as well just call getFieldsData and not use this at all. Seems unneccessary